static/app/components/assistant/guideAnchor.tsx

```diff
@@ -266,75 +223,9 @@ function GuideAnchor({disabled, children, ...rest}: WrapperProps) {
   return <BaseGuideAnchor {...rest}>{children}</BaseGuideAnchor>;
 }
 
-export default GuideAnchor;
-
-const GuideContainer = styled('div')`
-  display: grid;
-  grid-template-rows: repeat(2, auto);
-  gap: ${space(2)};
-  text-align: center;
-  line-height: 1.5;
-  background-color: ${p => p.theme.purple300};
-  border-color: ${p => p.theme.purple300};
-  color: ${p => p.theme.white};
-
-  a {
-    :hover {
-      color: ${p => p.theme.white};
-    }
-  }
-`;
-
-const GuideContent = styled('div')`
-  display: grid;
-  grid-template-rows: repeat(2, auto);
-  gap: ${space(1)};
-
-  a {
-    color: ${p => p.theme.white};
-    text-decoration: underline;
-  }
-`;
-
-const GuideTitle = styled('div')`
-  font-weight: ${p => p.theme.fontWeightBold};
-  font-size: ${p => p.theme.fontSizeExtraLarge};
-`;
-
-const GuideDescription = styled('div')`
-  font-size: ${p => p.theme.fontSizeMedium};
+const GuideAnchorWrapper = styled('span')`
+  display: inline-block;
+  width: 100%;
```

Comment:
These styles replicate the internals of `useHoverCard` overlay, returning a span with these styles.